### PR TITLE
More radio stations...

### PR DESCRIPTION
**Pull request by Xastreh**

### CODE REVIEW

File: constants.toml [hunk 1]
```diff
@@ -209,6 +214,7 @@
   "NonStopPlay"                      = "http://www.nonstopplay.com/site/media/pls/midband.pls"
   "Northsound 1"                     = "http://icy-e-bz-01-gos.sharp-stream.com/northsound1.mp3"
   "Nova"                             = "http://149.13.0.81/nova128.m3u"
+  "Nova 96.9"                        = "https://stream.novaentertainment.com.au/nova969"
```
Comment (zeyla) [inline constants.toml]: This one appears to be incompatible.

File: constants.toml [hunk 2]
```diff
@@ -87,6 +89,7 @@
   "CapitalFM XTRA London"            = "http://media-ice.musicradio.com/CapitalXTRALondonMP3"
   "Celestia Radio"                   = "http://celestia.aiverse.org:8000/mp3_256"
   "CeltCast"                         = "http://celtcast.gkstreamen.nl:8086/;listen.mp3"
+  "Coka-Cola FM Brazil"              = "http://ic.imusicaradios.com.br:8000/brasil.stream.http?url=tunein"
```
Comment (zeyla) [inline constants.toml]: Remove the `?url=tunein` please, doesn't seem required.

**Comment by Xastreh** [comment]
Alright, will make the changes

**Comment by Xastreh** [comment]
Sorry about the issues.

**Comment by zeyla** [comment]
Thanks

**Pull request merged**